README.md

```diff
@@ -4,18 +4,628 @@
 
 _visit www.actionherojs.com for more information_
 
-This tutorial showcases a realistic Actionhero project. In addition to the `actionhero` project, it uses the [`ah-seqeulzie-plugin`](https://github.com/actionhero/ah-sequelize-plugin) to use models and migrations against a Postgres database.
+This tutorial showcases a realistic Todo List Application with CRUD operations.
 
-## To install:
+To simply play around with the application, follow the next steps:
 
-You will need node.js, npm, redis, and postgres available.
+1. Make sure you've the following installed on your system
+    - Redis
+    - Postgresql 
+    - NodeJS
 
-`npm install`
+2. Clone the project
 
-## To Run:
+```
+git clone https://github.com/actionhero/actionhero-tutorial.git 
+&& cd actionhero-tutorial
+```
 
-`npm start`
+3. Install the dependencies
+
+```
+npm install
+```
+
+4. Configure the database connection in `./src/config/sequelize.js` and run migrations.
+```
+npx sequelize-cli db:migrate
+```
+
+5. Start Dev server and access API's at <a href="http://localhost:8080">Localhost</a>
+```
+npm run test actions/task
```

Comment:
What does this test command do?  Looks like just normal `npm test`?

Reply:
Yes, this just specifically tests `actions/task.ts`.
Since we are following the tutorial, we first test `models/task.ts` and then `actions/task.ts`